backend.go

```diff
@@ -145,15 +146,29 @@ func (b *jwtAuthBackend) jwtValidator(config *jwtConfig) (*jwt.Validator, error)
 
 	var err error
 	var keySet jwt.KeySet
+	var keySets []jwt.KeySet
 
 	// Configure the key set for the validator
 	switch config.authType() {
 	case JWKS:
 		keySet, err = jwt.NewJSONWebKeySet(b.providerCtx, config.JWKSURL, config.JWKSCAPEM)
+		keySets = []jwt.KeySet{keySet}
+	case MultiJWKS:
+		pairs, err := NewJWKSPairsConfig(config)
+
+		for _, p := range pairs {
+			keySet, keySetErr := jwt.NewJSONWebKeySet(b.providerCtx, p.JWKSUrl, p.JWKSCAPEM)
+			if keySetErr != nil {
+				err = multierror.Append(err, keySetErr)
+			}
+			keySets = append(keySets, keySet)
```

Comment:
I believe the error is shadowed here such that if errors occurred, they wouldn't be appended to the outer error. I don't think we'd end up in the `if err != nil {` check below this switch case.

Reply:
Good catch. Looks like we can rename that error to something like `pairsErr` and just check and return it before the loop if not nil.